### DISCUSSION TITLE
How to install from source with PNG delegates ?

### DISCUSSION THREAD

**Discussion by xiaodongzhang97** [ImageMagick/ImageMagick#4595] [Help]
I have successfully install ImageMagick from source, however I find there is no built-in PNG delegates.
I have tried some schemas in different forums.
But there is no one works for me.
I'm using ubuntu 18.
I hope to know if there is offical documents introducing how to install ImageMagick from sources with delegates.
Thanks very much!

**Comment by Danack** [comment]
You need to install the appropriate libraries -  [here's the list I use for debian](https://github.com/Imagick/imagick/blob/master/docker/developing/Dockerfile#L6-L13)

And then the [configure command](https://github.com/Imagick/imagick/blob/022ddd38f0731d554d106fd7cbfd664553dd057c/docker/installImageMagick.sh#L18-L30) to configure ImageMagick, before compiling.
